core/src/main/java/org/mule/runtime/core/internal/connection/ErrorTypeHandlerConnectionProviderWrapper.java

```diff
@@ -42,7 +42,7 @@ public ErrorTypeHandlerConnectionProviderWrapper(ConnectionProvider<C> connectio
                                                    ExtensionModel extensionModel, RetryPolicyTemplate retryPolicyTemplate) {
     super(connectionProvider);
     this.errorTypeRepository = muleContext.getErrorTypeRepository();
-    namespace = extensionModel.getXmlDslModel().getNamespace().toUpperCase();
+    namespace = extensionModel.getXmlDslModel().getPrefix().toUpperCase();
```

Comment:
not sure if you didn't see the commit [fixes after MGs review](https://github.com/mulesoft/mule/pull/4180/commits/01c58861231d7e49389197e63138a30452b2d9a2), but they weren't ignored